lib/ios/RNNModalManager.m

```diff
@@ -27,6 +27,9 @@ -(void)showModal:(UIViewController *)viewController animated:(BOOL)animated hasC
 	UIViewController* topVC = [self topPresentedVC];
 	topVC.definesPresentationContext = YES;
 	
+	UIViewController* topUIVC = [self topUIViewController:viewController];
+	topUIVC.modalPresentationStyle = topVC.modalPresentationStyle;
```

Comment:
Maybe more properties need to be applied or we need a more generic way to apply other properties.